src/Uno.UWP/UI/Notifications/ToastNotifier.Android.cs

```diff
@@ -0,0 +1,280 @@
+﻿#if __ANDROID__
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace Windows.UI.Notifications
+{
+	public partial class ToastNotifier
+	{
+		private string _channelID = "PlatformUno";

```

Comment:
What would happen if two different uno apps use the same channelId ? Would that make some conflicts?

Reply:
Doc says: "The id of the channel. Must be unique per package. ". It seems to resolve your question.